src/api/app/assets/stylesheets/webui/application/style.scss

```diff
@@ -412,3 +412,15 @@ div.dataTables_wrapper:after {
 #subheader #breadcrump {
   line-height: 1.3em;
 }
+
+.outdated_buildresult {
```

Comment:
shouldn't all this styles be in [package.scss](https://github.com/mdeniz/open-build-service/blob/master/src/api/app/assets/stylesheets/webui/application/package.scss)? :thinking: